### ISSUE TITLE
[Forge] Forge support is failing

### ISSUE THREAD

**Issue by samip5** [SpigotMC/BungeeCord#2612]
Forge version: 14.23.5.2781
Modpack: Enigmatica2-1.62 (MC: 1.12.2)

Says invalid session, even when it surely is not.

Log says:
```
<PUBLIC-IP-REDACTED> <-> InitialHandler - NativeIoException: syscall:read(..) failed: Connection reset by peer
```

Any ideas what would be needed to fix it myself? I know Java.

**Comment by Janmm14** [comment]
This error indicates the connection was closed by the otber side of the connection

**Comment by samip5** [comment]
@Janmm14 Yes,  that much I know but why and how I should be able to troubleshoot it?  Plus maybe fix it?

 My Forge server is not running SpongeForge,  as it requires newer Forge than the available one with the Sponge version.
